SQL_DDL_Nino_Jintcharadze_HW.sql.sql

```diff
@@ -0,0 +1,164 @@
+CREATE SCHEMA sushi_delivery_service;
+
+CREATE TABLE Customer(
```

Comment:
Use schema_name.table_name because the same table_name can exists in other schema